samples/tfm_integration/tfm_ipc/boards/nucleo_l552ze_q_ns.overlay

```diff
@@ -28,18 +31,18 @@
 
 		boot_partition: partition@0 {
 			label = "mcuboot";
-			reg = <0x00000000 0x00011000>;
+			reg = <0x00000000 0x00013000>;
 			read-only;
 		};
 		/* Secure image primary slot */
-		slot0_partition: partition@00011000 {
+		slot0_partition: partition@00013000 {
 			label = "image-0";
-			reg = <0x00011000 0x00040000>;
+			reg = <0x00013000 0x00038000>;
 		};
 		/* Non-secure image primary slot */
-		slot1_partition: partition@00051000 {
+		slot1_partition: partition@0004B000 {
 			label = "image-1";
-			reg = <0x00051000 0x0002E000>;
+			reg = <0x0004B000 0x0002A000>;
```

Comment:
Is that required ?
If yes, I think I'd be better moved in dedicated commit anyway

Reply:
There is already a dedicated commit, @erwango concerning this, in the trusted-firmware-m repository.
https://github.com/zephyrproject-rtos/trusted-firmware-m/pull/15/commits/8dca035c06a23907998c2c550f55fdc68136bc86

Reply:
So this is required, because the TFM layout is changed now (or, strictly speaking , it will be changed if the upstream fixes get merged). I can move this to a separate commit, no problem.

Reply:
moved, now.